ngraph/python/tests/mock/pyngraph_fe_mock_api/CMakeLists.txt

```diff
@@ -11,6 +11,7 @@ source_group("src" FILES ${PYBIND_FE_SRC})
 
 pybind11_add_module(${PYBIND_FE_NAME} MODULE ${PYBIND_FE_SRC})
 
-target_link_libraries(${PYBIND_FE_NAME} PRIVATE ${TARGET_FE_NAME})
+target_link_libraries(${PYBIND_FE_NAME} PRIVATE ${TARGET_FE_NAME} ngraph::frontend_manager::static)
+target_compile_definitions(${PYBIND_FE_NAME} PRIVATE USE_STATIC_FRONTEND_MANAGER)
```

Comment:
can we put this definition as `PUBLIC` for `ngraph::frontend_manager::static`? In this case we don't need to duplicate this line in each usage